Exiled.API/Exiled.API.csproj

```diff
@@ -38,7 +38,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp-Publicized">
-      <HintPath>$(EXILED_REFERENCES)\Assembly-CSharp-Publicized.dll</HintPath>
+      <HintPath>..\Refs\Assembly-CSharp-Publicized.dll</HintPath>
```

Comment:
All references should use $(EXILED_REFERENCES) - the assembly provided in /Refs is only for those who need the publicized assembly for their own plugin projects.

Reply:
Why do I need to duplicate assembly in `$(EXILED_References)`, if it can be imported from `Refs/`, I see this as superfluous action, if we provide the latest `Assembly-CSharp-Publicized.dll`, then why move it every update?

Reply:
Consistency with all the other references?

Reply:
Developer must have a complete set of updated refs according to the freshness of `Assembly-CSharp-Publicized.dll`?